python/Makefile.am

```diff
@@ -8,22 +8,22 @@ if WITH_MPI
   MDPYTEST=
   MPBPYTEST=
 else
-  MDPYTEST=$(TEST_DIR)/material_dispersion.py
+  MDPYTEST=$(TEST_DIR)/test_material_dispersion.py
```

Comment:
It shouldn't be necessary to start every file with `test_*.py` — they are in a "tests" directory after all.

Reply:
In any case, it just makes it harder to review the PR if it mixes multiple unrelated changes.

Reply:
The use of the `test_` prefix for files and test functions is a pytest convention.
It makes it easy to distinguish actual test cases and functions from helpers such as those in `utils.py`.
Pytest is so widely used across Python projects, that most developers are likely confused by the lack of the `test_` prefix.

I am going to split this PR to facilitate reviews.